TestApps/Sasquatch/SasquatchSwift/AppDelegate.swift

```diff
@@ -38,20 +38,11 @@ class AppDelegate: UIResponder, UIApplicationDelegate, MSCrashesDelegate, MSDist
       MSAppCenter.startFromLibrary(withServices: [MSAnalytics.self])
       break;
     case startFrom.APP.rawValue:
-      #if DEBUG
-        MSAppCenter.start(secretString, withServices: [MSAnalytics.self, MSCrashes.self, MSPush.self])
-      #else
-        MSAppCenter.start(secretString, withServices: [MSAnalytics.self, MSCrashes.self, MSDistribute.self, MSPush.self])
-      #endif
+      MSAppCenter.start(secretString, withServices: [MSAnalytics.self, MSCrashes.self, MSDistribute.self, MSPush.self])
```

Comment:
You are removing the
```swift
#if DEBUG
#else
#endif``` because you want to test them during automated UI testing?

Reply:
I just realized that Puppet didn't use the macro at all, so I think that's okay to adjust this

Reply:
Yes. We have a test that verify enable/disable distribute module, so it should be started to make test work